.github/pull_request_template.md

```diff
@@ -3,6 +3,8 @@ DEVPROD-NNNNN
 ### Description
 add description, context, thought process, etc
 
+<!-- Are you adding a Task, Build, Version, or Patch struct? Create a DPIPE ticket to expose this in data warehouse. -->
```

Comment:
Do you mean adding a field to one of those structs?

Reply:
omg yes good catch